app/src/main/java/ml/docilealligator/infinityforreddit/adapters/HistoryPostRecyclerViewAdapter.java

```diff
@@ -2853,11 +2907,24 @@ public void release() {
                 helper.release();
                 helper = null;
             }
+            isManuallyPaused = false;
+            container = null;
         }
 
         @Override
         public boolean wantsToPlay() {
-            return canPlayVideo && mediaUri != null && ToroUtil.visibleAreaOffset(this, itemView.getParent()) >= mStartAutoplayVisibleAreaOffset;
+            if (canPlayVideo) {
+                if (ToroUtil.visibleAreaOffset(this, itemView.getParent()) >= mStartAutoplayVisibleAreaOffset) {
+                    if (isManuallyPaused) {
+                        play();
```

Comment:
Why is `play()` needed here?

Reply:
Without it, helper.setVolume causes a crash after a release() (say when when you scroll the player out of view), I think it's because play() makes some initializations.